CHANGELOG_iOS.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## [1.69.170](https://github.com/brave/brave-browser/releases/tag/v1.69.170)
+
+### Web3
+
+ - Enabled Bitcoin by default in Brave Wallet. ([#38775](https://github.com/brave/brave-browser/issues/38775))
+
+### General
+
+ - Added per-site adblocking in the Brave Shields panel. ([#39573](https://github.com/brave/brave-browser/issues/39573))
+ - Added advanced server selection for Brave VPN. ([#36409](https://github.com/brave/brave-browser/issues/36409))
+ - Added country server details to the VPN “Server Region” page. ([#39824](https://github.com/brave/brave-browser/issues/39824))
+ - Improved standard mode shields logic to allow some ads that were being hidden. ([#40967](https://github.com/brave/brave-browser/issues/40967))
+ - Updated model names in the Leo menu. ([#40464](https://github.com/brave/brave-browser/issues/40464))
+ - Updated adblock-rust to v0.8.11 which applies “removeparam” to document/subdocument/xhr by default. ([#36039](https://github.com/brave/brave-browser/issues/36039))
+ - Updated Leo input box to match recent design changes. ([#38960](https://github.com/brave/brave-browser/issues/38960))
+ - Updated favorite name on New Tab Page from “Twitter” to “X”. ([#39322](https://github.com/brave/brave-browser/issues/39322))
```

Comment:
I don't think we need this one in the release notes TBH. Lets just label it `release-notes/exclude` and remove it.